systemtest/pom.xml

```diff
@@ -42,6 +42,11 @@
             <artifactId>hamcrest-all</artifactId>
             <version>${hamcrest.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path-assert</artifactId>
+            <version>2.2.0</version>
```

Comment:
Same as above ... is the scope really `compile`?

Reply:
@scholzj  it is `test`